apps/frontend/src/app/Data/Characters/KujouSara/index.tsx

```diff
@@ -275,6 +275,7 @@ const sheet: ICharacterSheet = {
         value: condC6,
         path: condC6Path,
         teamBuff: true,
+        canShow: equal(condSkillTenguAmbush, 'TenguJuuraiAmbush', 1),
```

Comment:
If we are tying this buff to this conditional, then you also need to update the buff logic on line 94